components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/MeApi.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.v1;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.v1.MeApiService;
+import org.wso2.carbon.apimgt.rest.api.publisher.v1.impl.MeApiServiceImpl;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.inject.Inject;
+
+import io.swagger.annotations.*;
+import java.io.InputStream;
+
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import java.util.Map;
+import java.util.List;
+import javax.validation.constraints.*;
+@Path("/me")
+
+@Api(description = "the me API")
+@Consumes({ "application/json" })
+@Produces({ "application/json" })
+
+
+public class MeApi  {
+
+  @Context MessageContext securityContext;
+
+MeApiService delegate = new MeApiServiceImpl();
```

Comment:
Formatting issue. Shall we add a tab space before starts the line

Reply:
This file contains auto-generated code..